packages/server/graphql/public/rules/isOrgTier.ts

```diff
@@ -2,13 +2,24 @@ import {rule} from 'graphql-shield'
 import {GQLContext} from '../../graphql'
 import {TierEnum} from '../resolverTypes'
 
-const isOrgTier = (requiredTier: TierEnum) =>
-  rule({cache: 'strict'})(async (_source, {orgId}, {dataLoader}: GQLContext) => {
-    const organization = await dataLoader.get('organizations').load(orgId)
-    if (!organization) return new Error('Organization not found')
-    const {tier} = organization
-    if (tier !== requiredTier) return new Error(`Organization is not ${requiredTier}`)
-    return true
-  })
+const resolve = async (requiredTier: TierEnum, orgId: string, {dataLoader}: GQLContext) => {
+  const organization = await dataLoader.get('organizations').load(orgId)
+  if (!organization) return new Error('Organization not found')
+  const {tier} = organization
+  if (tier !== requiredTier) return new Error(`Organization is not ${requiredTier}`)
+  return true
+}
 
-export default isOrgTier
+export const isOrgTierSource = (requiredTier: TierEnum) =>
```

Comment:
Looks fine. `isSourceOrgTier` would read more fluently, but it wouldn't work with `isViewerBillingLeader`

Reply:
naming is hard 😖. totally agree! My hope is a general suffix will make it easier to catch the error. e.g. I copied these rules from 'updateSAML` and they didn't work because they were pulling from `args`. Hopefully as permissions grow, it'll be easy to say "Oh, most of the rules on a type object end in `Source`, whereas there is no suffix on rules on `Mutation, Query`.